cluster-provision/okd/scripts/run.sh

```diff
@@ -70,6 +79,8 @@ done
 # wait half minute, just to be sure that we do not get old cluster state
 sleep 30
 
+uncordon_nodes
```

Comment:
hm do we still need to uncordon workers, also with 2 workers on install time?

Reply:
That's right, also I am going to wait at scale down at provision.

Reply:
I still can see uncordon methods under the run, can you please re-check?